scripts/utils/TagNuGetClientGitRepostoryOnRelease.ps1

```diff
@@ -48,7 +48,7 @@ if($index -ne '-1')
 
 $Date = Get-Date
 $Message = "Insert $ProductVersion into $VsTargetBranch on $Date"
-$BuildInfoJsonFile = [System.IO.Path]::Combine($BuildOutputPath, $Branch, $Build, 'buildinfo.json')
+$BuildInfoJsonFile = [System.IO.Path]::Combine('$(System.ArtifactsDirectory)','$(Build.DefinitionName)','BuildInfo','buildinfo.json')
 $buildInfoJson = (Get-Content $BuildInfoJsonFile -Raw) | ConvertFrom-Json
```

Comment:
Where are those `Build.DefinitionName` etc things defined? Are they some props or environmental vars?

Reply:
@erdembayar https://docs.microsoft.com/en-us/azure/devops/pipelines/release/variables?view=azure-devops&tabs=batch

@kartheekp-ms I don't think this is the right syntax. It works in a build's inline script, because Azure Pipelines will replace the variable in the script before saving it in a `.ps1` file. At the bottom of the doc I linked:

>To use a default variable in your script, you must first replace the . in the default variable names with _. For example, to print the value of artifact variable Release.Artifacts.{Artifact alias}.DefinitionName for the artifact source whose alias is ASPNET4.CI in a PowerShell script, you would use $env:RELEASE_ARTIFACTS_ASPNET4_CI_DEFINITIONNAME.

However, I'd prefer if the buildinfo.json path/filename was passed in as a parameter to the script instead (lines 20-26)

Reply:
@zivkan 
Follow up question? How to you debug or test? Can we do on your local or have to deploy to test?

Reply:
@zivkan - Thank you for the review comment. I modified `TagNuGetClientGitRepostoryOnRelease.ps1` to now accept `BuildInfoJson` file as parameter.

I propose changes to the following tasks in `NuGet Client Release Pipeline`
- `Insert into VS (Get VS Target Branch task) `
- `Insert into VS (Get Assembly Version) `
- `Insert into VS (Tag NuGet.Client Git repository) ` - Pass `$BuildInfoJsonFile` as parameter to [TagNuGetClientGitRepostoryOnRelease.ps1](https://github.com/NuGet/NuGet.Client/blob/dev/scripts/utils/TagNuGetClientGitRepostoryOnRelease.ps1)script.

```ps1
$BuildInfoJsonFile = [System.IO.Path]::Combine('$(System.ArtifactsDirectory)','$(Release.PrimaryArtifactSourceAlias)','BuildInfo','buildinfo.json') 
```
[$(Release.PrimaryArtifactSourceAlias)](https://docs.microsoft.com/en-us/azure/devops/pipelines/release/variables?view=azure-devops&tabs=batch#default-variables---release) is the alias of the primary artifact source. I am leaning towards using the default alias because we have only one primary artifact source instead of specifying the source alias `$(Release.Artifacts.NuGet.Client-Official.DefinitionName)` explicitly.